### ISSUE TITLE
Longitude PFD touch screen not working

### ISSUE THREAD

**Issue by Keiferz** [hawkeye-stan/msfs-popout-panel-manager#105]
I recently downloaded pop out manager 4.0.3.0004 to use an Android phone as the PFD using spacedesk on the Citation Longitude.  I'm using spacedesk version 2.1.0.9 as the desktop viewer.  The popout works perfectly but I'm havin an issue with the touchscreen not working.  I have tried changing touch settings in both applications. For spacedesk I have tried disable/enable touchscreen as well as using absolute touch (would assume this is what is needed) and relative touch (mimics mouse pad). Absoulte touch, I can see the PFD buttons highlight but they don't actually do anything. I have tried all touchscreen delay settings from 0 to 100ms with the same issue.  
I have tried deleting cache off of MSFS as well and re-installed MSFS 2020, spacedesk and popout Manager with the same issue.

Is anyone else having this issue using spacedesk?
Does anyone have any other remote desktop viewer that does work with MSFS Pop Out Panel Manager for touchscreen?
Is there some other setting I'm not seeing?

TO recap versions being used:
-Pop Out Manager 4.0.3.0004
-SpaceDesk version 2.1.0.9
-Galaxy S23 Ultra - Android Version 14 - One UI 6.0 - Kernal version 5.15.94
-MSFS 2020 1.35.21.0 

Any help all would be much appreciative. Thanks!

**Comment by Keiferz** [comment]
EDIT:  OK, relative does work I just had to click twice to get it to select the button press. I'm currently using this as a workaround as I still have issues with absolute touch (touchscreen).

Toggling panel touch capability has the same charateristics whether it's enabled or disabled in popout manager.

After more testing I found there is a few buttons that I did get to work briefly but only for a short time

Buttons that I saw working with touchscreen (absolute):

Center pedestal (Pilot Side)
From Home Screen:
-Waypoint Info
    Create Waypoint

-Nearest
     Weather

When clicking around with the mouse on other screens seems to break those that did work.  I'm guessing this is an issue with the touch portion not aligning the mouse properly on the PFD or not actually registering the release part of the touch. The buttons do highlight for all buttons when clicking on them but most don't work at all and if they do it's completely random.

**Comment by Keiferz** [comment]
RESOLVED:

I guess I didn't troubleshoot enough before opening this thread but maybe this will be useful for anyone else experiencing issues.

In spacedesk I was using native resolution.  After changing to custom resolution there are few resolutions that do not work and more that do.  If you're having issues with the touchscreen not working try messing around with the resolution in spacedesk as well as display settings in Windows and I'm sure you will get it working. 

Hope this helps anyone else with touchscreen issues using pop out manager.

Thank you Stanley for this most beautiful product!

**Comment by hawkeye-stan** [comment]
Thanks, Keiferz, you're a genius and so glad you found the solution to your issue. 

Looks like Spacedesk touch emulation and resolution are somehow related. That's probably why some users reported that on one android device Spacedesk touch works and another does not work and they're practically should be the same. Until now, I've no clue why that's the case.
